### PR TITLE
feat: add url property and clean up trigger resources

### PR DESCRIPTION
**Pull request by ianaya89**
## Affected Components
* [x] Resources
* [ ] Tests
* [ ] Docs
* [ ] Other

## Style
* [x] Terraform code is formatted with `terraform fmt`
* [x] Go code is formatted with `go fmt`

## Notes for the Reviewer

- Clean up check/group triggers resources
- Add `URL` property to triggers

> Depends on https://github.com/checkly/checkly-go-sdk/pull/63

### CODE REVIEW

**Comment by ianaya89** [comment]
@danielpaulus can you help me with a review?

File: checkly/resource_trigger_group.go [hunk 1]
```diff
@@ -71,58 +75,46 @@ func resourceTriggerGroupCreate(d *schema.ResourceData, client interface{}) erro
 		return fmt.Errorf("CreateTriggerGroup: API error: %w", err)
 	}
 
-	d.SetId(fmt.Sprintf("%d", result.ID))
-	d.Set("token", result.Token)
```
Comment (danielpaulus) [inline checkly/resource_trigger_group.go:75]: is that ok to delete? It seems it is being used further down the line.

Comment (ianaya89) [reply checkly/resource_trigger_group.go:75]: It's being set by `resourceDataFromTriggerGroup` later

Comment (ianaya89) [reply checkly/resource_trigger_group.go:75]: It's being set by resourceDataFromTriggerGroup later

**Review by danielpaulus — APPROVED** [review]

**Pull request merged**